app/assets/javascripts/components/shared/modal.js

```diff
@@ -24,6 +24,11 @@
         showTitle: false,
         // Title of the modal – mandatory for accessibility
         title: 'Untitled modal',
+        // Allows the modal container have scroll
+        allowScroll: false,
```

Comment:
Why did you add that? We didn't need it before and you're not using it for the save modal.